radix-engine/src/engine/modules/module.rs

```diff
@@ -83,58 +83,72 @@ pub enum SysCallOutput<'a> {
 pub trait Module<R: FeeReserve> {
     fn pre_sys_call(
         &mut self,
-        call_frame: &CallFrame,
-        heap: &mut Heap,
-        track: &mut Track<R>,
-        input: SysCallInput,
-    ) -> Result<(), ModuleError>;
+        _call_frame: &CallFrame,
```

Comment:
This is very unconventional. Why do we need to add `_` to parameters in traits?

Reply:
Added default implementations, so `_` used to show that these parameters aren't used in the default implementation.

Reply:
You mean IDE generated default implementations? Should we just change the implementation if it's not using the arguments, rather than the traits? 

My concer is that it will disable unused varilable checks, and makes documentation "ugly".

Reply:
Not IDE generated, but the Rust trait default implementation (you can see all of these returning `Ok(())`).

Reply:
I see.